src/kvstore/raftex/test/TestShard.h

```diff
@@ -174,21 +174,19 @@ class SnapshotManagerImpl : public SnapshotManager {
                           << ", total count sended " << totalCount
                           << ", total size sended " << totalSize
                           << ", finished false";
-                cb(std::move(data), totalCount, totalSize, false);
+                cb(std::move(data), totalCount, totalSize, SnapshotStatus::IN_PROGRESS);
                 data.clear();
             }
             auto encoded = encodeSnapshotRow(row.first, row.second);
             totalSize += encoded.size();
             totalCount++;
             data.emplace_back(std::move(encoded));
         }
-        if (data.size() > 0) {
```

Comment:
seems should not remove this check ?

Reply:
We remove the if condition to ensure the callback function always being called in any case.